packages/web-pkg/src/components/sideBar/Details/SpaceDetails.vue

```diff
@@ -77,11 +77,16 @@ import SpaceQuota from '../../SpaceQuota.vue'
 import { formatDateFromISO } from 'web-pkg/src/helpers'
 import { configurationManager } from 'web-pkg/src/configuration'
 import { eventBus } from 'web-pkg/src/services/eventBus'
-import { SideBarEventTopics } from '../../../composables/sideBar'
 
 export default defineComponent({
   name: 'SpaceDetails',
   components: { SpaceQuota },
+  props: {
+    spaceResource: {
+      type: Object,
```

Comment:
Please use the correct type: 
```js
type: Object as PropType<SpaceResource>
```
Also: any reason why you didn't just name the variable `space`?

Reply:
because space is already used ^^

Reply:
yeah ok, sorry 😆 then keep the name like it is. But please specify the type.

Reply:
yes